notifications_utils/version.py

```diff
@@ -5,4 +5,4 @@
 # - `make version-minor` for new features
 # - `make version-patch` for bug fixes
 
-__version__ = '51.2.0'  # 1e1f0a68e40b7c2b164567e4d7e1d373
+__version__ = '51.1.1'  # 5fd0bbc81c3653cee9fa2ada1b78c8c5
```

Comment:
Do we actually need to release this as something above 51.2.0. I don't know if our pipeline will have an issue with trying to republish lower than the current version?

Reply:
Oops, thanks for spotting. You're right, we shouldn't do this: there's already a tag for 51.1.1. 

I've amended the commit to a higher, patch version.